src/librustc/mir/mod.rs

```diff
@@ -22,7 +22,7 @@ use mir::visit::MirVisitable;
 use rustc_apfloat::ieee::{Double, Single};
 use rustc_apfloat::Float;
 use rustc_data_structures::control_flow_graph::dominators::{dominators, Dominators};
-use rustc_data_structures::control_flow_graph::ControlFlowGraph;
+use rustc_data_structures::control_flow_graph;
```

Comment:
ultra-nit: shouldn't this just be a `self` added to the `use` on the the next line?